### PR TITLE
Getting started run test

### PR DESCRIPTION
**Pull request by bdharris08**
#57

Open to changes.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -125,6 +127,20 @@ Back in the terminal:
   1 passing (9ms)
 ```
 
+Set up a test script in package.json:
+
+```json
+"scripts": {
+    "test": "./node_modules/mocha/bin/mocha"
```
Comment (Munter) [inline index.md]: npm script implicitly have `node_modules/.bin` in `$PATH`. It's enough to just write `"mocha"` here

Comment (bdharris08) [reply index.md]: Good point, thanks. I committed that change to the PR.

**Comment by boneskull** [comment]
@bdharris08 Thanks.  Can you please [sign the CLA](https://cla.js.foundation/mochajs/mochajs.github.io?pullRequest=58)?

**Comment by boneskull** [comment]
I'll implement this myself later this evening if the CLA in this or #53 is not signed; this causes a lot of confusion

**Comment by bdharris08** [comment]
I should have read the contributing guide, sorry for the confusion. If a merge is needed I'll do it.

**Comment by boneskull** [comment]
@bdharris08 There's no up-to-date contributing guide for Mocha (yet), though it's in progress: #61 

Thanks!

**Comment by bdharris08** [comment]
@boneskull Ok, I signed the CLA. Checked on #53 and it's essentially the same change except for the npm test script in the package.json. Let me know if you would like any other changes.

**Comment by STRAYKR** [comment]
Sorry for not responding...thanks for making the change; much better now. :+1:

**Pull request merged**